cookbooks/2D_subduction_with_two_phase_flow/fixed_slab.prm

```diff
@@ -0,0 +1,278 @@
+    set Fluid compressibility                            = 0
+    set Fluid reaction time scale for operator splitting = 1e4
+    set Fluid-solid reaction scheme                      = tian approximation
+    subsection Tian 2019 model
+      set Maximum weight percent water in peridotite       = 4
+      set Maximum weight percent water in gabbro           = 2
+      set Maximum weight percent water in MORB             = 3
+      set Maximum weight percent water in sediment         = 4
+    end
+  end
+
+  subsection Visco Plastic
+    set Viscosity averaging scheme                        = geometric
+    set Viscous flow law                                  = composite
+    set Viscosity prefactor scheme                        = HK04 olivine hydration
+    set Minimum strain rate                               = 1e-20
+    set Reference strain rate                             = 1e-20
+
+    set Phase transition depths           =   background: 660e3,  porosity: 660e3, bound_fluid: 660e3, peridotite: 660e3, gabbro: 660e3, MORB: 660e3, sediment: 125e3|660e3
+    set Phase transition widths           =   background: 1e3,    porosity: 1e3,   bound_fluid: 1e3,   peridotite: 1e3,   gabbro: 1e3,   MORB: 1e3,   sediment: 10e3|1e3
+    set Phase transition temperatures     =   background: 1573,   porosity: 1573,  bound_fluid: 1573,  peridotite: 1573,  gabbro: 1573,  MORB: 1573,  sediment: 1573|1573
+    set Phase transition Clapeyron slopes =   background: 0,      porosity: 0,     bound_fluid: 0,     peridotite: 0,     gabbro: 0,     MORB: 0,     sediment: 0|0
+
+    # set Densities                         =   background: 3300,   porosity: 3300,  bound_fluid: 3300,  peridotite: 3300,  gabbro: 3100,  MORB: 3000,  sediment: 2900
+    set Densities                         =   background: 3300,   porosity: 3300,  bound_fluid: 3300,  peridotite: 3300,  gabbro: 3300,  MORB: 3175,  sediment: 3175
+
+    set Minimum viscosity   =   background: 2.5e18|2.5e18,  porosity: 2.5e18|2.5e18, bound_fluid: 2.5e18|2.5e18, peridotite: 2.5e18|2.5e18, gabbro: 2.5e18|2.5e18, MORB: 2.5e18|2.5e18, sediment: 9.999e18|2.5e18|2.5e18
+    set Maximum viscosity   =   background: 2.5e24|2.5e23,  porosity: 2.5e23|2.5e23, bound_fluid: 2.5e23|2.5e23, peridotite: 2.5e23|2.5e23, gabbro: 2.5e23|2.5e23, MORB: 2.5e23|2.5e23, sediment: 1.001e19|2.5e23|2.5e23
+
+
+    set Prefactors for diffusion creep                    = background:  1.577e-21, \
+                                                            porosity:    1.577e-21, \
+                                                            bound_fluid: 1.577e-21, \
+                                                            peridotite:  1.577e-21, \
+                                                            gabbro:      1.577e-21, \
+                                                            MORB:        1.577e-21, \
+                                                            sediment:    1.577e-21
```

Comment:
I might be wrong about this, but if they're all the same, can't you just list them as 
```suggestion
    set Prefactors for diffusion creep                    = 1.577e-21
```